### ISSUE TITLE
Unable to complete market buy orders & limit orders not completing

### ISSUE THREAD

**Issue by bitstashco** [0xProject/0x-launch-kit#22]
As you can see from the below attached screenshot. The error from console. Market sells work as expected. Limit orders also do not match as expected. The only thing currently working as expected is the market sell for orders. 

![Screenshot from 2019-07-02 16-31-50](https://user-images.githubusercontent.com/39512094/60553289-7a9d0500-9ce7-11e9-8c95-d36085b59d48.png)

Note: limit orders will stay/show on the exchange however the spread is incorrectly displayed. Please check the second attached image for reference.

![Screenshot from 2019-07-02 16-43-48](https://user-images.githubusercontent.com/39512094/60553572-c00e0200-9ce8-11e9-9cbe-7306e6c696ec.png)

**Comment by dekz** [comment]
> Limit orders also do not match as expected

Can you expand on this, what behaviour are you expecting?

**Comment by bitstashco** [comment]
Yes, as you can see in the second image there is one buy limit and one sell limit order that has been placed for 0.001 weth and 1 zrx.

These orders are not being filled automatically as I would expect.

**Comment by dekz** [comment]
@bitstashco this automated matching is not a current feature for the Open Orderbook model. As it crosses a boundary of active participation, an activity some teams desire to avoid.

In the real world, the opportunity for profit attracts actors in ecosystem to match, rather than the operator of Launch Kit.

**Comment by bitstashco** [comment]
I understand, we are simply testing how things work with 0x. The primary issue was that market buys were not going through and giving an error on the initial install.

**Comment by bitstashco** [comment]
As to an update to the issue. I have tested with 0 fees and everything works as expected. There for some reason are some issues when setting a fee amount on kovan for market buy orders. 

I guess this is a similar issue with the back-end as well as it mentions:
RPC_URL -- Update with your node url. NOTE: Kovan doesn't work on INFURA with the current version of the OrderWatcher.

Which could be why the order produces an error. I do appreciate your fast response and time helping.

**Comment by dekz** [comment]
`RPC_URL -- Update with your node url. NOTE: Kovan doesn't work on INFURA with the current version of the OrderWatcher.` Oops this is now out of date as INFURA got an upgrade!

I'll try and reproduce what you're seeing. Did you deposit ETH into WETH (there's two flows depending if the user has WETH or only ETH)? Were there any fees on the orders?

**Comment by bitstashco** [comment]
Yes I deposited ETH and converted it to WETH I also got ZRX as well. I then split the balances into 3 addresses for testing. Fee address #1 and trading accounts #2 and #3.

I have tested on Kovan and Ropsten the error has occurred on both test networks.

I set the maker and takers fees to 0.01 ZRX.

Everything was left default except for the eth address for fees. 

Regarding the flows, on Kovan & Ropsten I tested with converted eth (weth) and without for the market buys and it produced the same error.
